### ISSUE TITLE
Can't change the displayed field in a linked table when a filter is applied

### ISSUE THREAD

**Issue by jajwilson** [ox-it/ords#723]
1. Filter the Mondial database so that, for instance, it displays only cities with greater than one million inhabitants.
2. Try changing the 'country' field to display country names rather than codes.
3. Watch as the processing symbol grinds round without any screen update every actually occuring

Chrome, Dev 1.0.16 - 4622

**Comment by jajwilson** [comment]
Upping this to critical as it's causing embarrassment in demonstrations!

**Comment by scottbw** [comment]
I've checked in a fix for this - it was a case of a query not being normalised

(the RQB filter uses aliases such as "x0", "x1" etc for table names, I have to change these back to actual table names before merging into other queries. I'd missed one of the patterns I needed to check for. I've added another unit test to catch this.)

**Comment by jajwilson** [comment]
That's odd, I'm still finding the same problem. The processing symbol still just grinds around indefinitely.

**Comment by jajwilson** [comment]
Ah, but in another database (one with 3+ linked tables) it DOES seem to work. Let me try again.

**Comment by jajwilson** [comment]
Yes - this does seem to have been fixed for most databases, but I'm still having problems with the original example db. When I filter the 'city' table in the 'Mondial again' db in the 'Standard test' project I can't get the filtered results to show the 'country name' field rather than country 'code'. Very puzzling. I'm wondering if this is just because this database is an old example and this change is for some reason only being applied to new databases.

**Comment by thestoat** [comment]
Is this the same problem as 579?

**Comment by jajwilson** [comment]
No. They're quite different. Did you mean 579 or is that a typo?

**Comment by thestoat** [comment]
Fat finger syndrome. I meant 759?

**Comment by jajwilson** [comment]
It's closely related, but not quite the same. In this ticket the symptom described in #759 is also the problem, but it was only occurring after a filter had been applied. In #759 it's worse - the symptom occurs even when no filter has been applied.
